docs/developer_guide/building_documentation.rst

```diff
@@ -10,13 +10,13 @@ First, create a clean environment and type the following commands in your termin
 
   git clone https://github.com/catalystneuro/neuroconv
   cd neuroconv
-  pip install -e .[docs]
+  pip install --editable .[docs]
```

Comment:
Why --editable instead of -e?

Reply:
More explicit commands are better for documentation. -e is the shorthand for editable which is good in interactive sessions.